dev-qt/assistant/assistant-5.15.9999.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
 QT5_MODULE="qttools"
-inherit desktop qt5-build xdg-utils
+inherit desktop qt5-build xdg-utils kde.org
```

Comment:
I was wondering why `kde.org` is inherited by every single ebuild instead of by `qt5-build.eclass`?

Reply:
Can do, although every single ebuild it is not - e.g. dev-qt/qtwebengine is still pulled from qt.io, and not every module got changes pushed to invent.kde.org yet. It was just a way to make sure none of the existing ebuilds is affected by the eclass changes.

I suppose kde.org could be conditionally inherited if `KDE_ORG_COMMIT` is set.

Reply:
ok I see, I'm fine either way.

Reply:
Went a step further and made `qt5-build.eclass` fully embrace `kde.org.eclass` by dropping `src_unpack()` phase. I like that solution now much better.